studio/components/interfaces/Auth/RedirectUrls/RedirectUrls.tsx

```diff
@@ -72,20 +84,25 @@ const RedirectUrls = () => {
     // Remove selectedUrl from array and update
     const payload = URI_ALLOW_LIST_ARRAY.filter((e: string) => e !== url)
 
-    const { error } = await authConfig.update({ URI_ALLOW_LIST: payload.toString() })
-
-    if (!error) {
-      setSelectedUrlToDelete(undefined)
-      ui.setNotification({ category: 'success', message: 'Successfully removed URL' })
-    } else {
-      ui.setNotification({
-        error,
-        category: 'error',
-        message: `Failed to remove URL: ${error?.message}`,
-      })
-    }
-
-    setIsDeleting(false)
+    updateAuthConfig(
+      { projectRef: projectRef!, config: { URI_ALLOW_LIST: payload.toString() } },
+      {
+        onError: (error) => {
```

Comment:
likewise as per https://github.com/supabase/supabase/pull/16227/files#r1289813556
shift all these to the mutation declaration

use isLoading from the mutation rather than isSubmitting from the Form

Reply:
Can't, due to `resetForm`.  I've removed `isSubmitting`.